doc/README.txt

```diff
@@ -11,33 +11,63 @@ Organization
 
 This is the top level build directory for the Matplotlib
 documentation.  All of the documentation is written using sphinx, a
-python documentation system built on top of ReST.  This directory contains
+python documentation system built on top of ReST.  This directory contains the
+following
 
-* users - the user documentation, e.g., installation, plotting tutorials,
-configuration tips, faq, explanations, etc.
+Files
+^^^^^
 
-* devel - documentation for Matplotlib developers
+* index.rst - the top level include document (and landing page) for the Matplotlib docs
 
-* api - placeholders to automatically generate the api documentation
+* conf.py - the sphinx configuration
 
-* tutorials, plot_types, and gallery - automatically
-generated by sphinx-gallery from ``../tutorials``, ``../plot_types``, and
-``../examples`` respectively (these are only present if docs have been
-built locally).
+* docutils.conf - htmnl output configuration
 
-* thirdpartypackages - redirect to <https://matplotlib.org/mpl-third-party/>
+* Makefile and make.bat - entry points for building the docs
 
-* mpl_toolkits - documentation of individual toolkits that ship with
-  Matplotlib
+* matplotlibrc - rcParam configuration for docs
 
-* index.rst - the top level include document for Matplotlib docs
+* missing-references.json - list of known missing/broken references
 
-* conf.py - the sphinx configuration
 
-* Makefile and make.bat - entry points for building the docs
+Content Folders
+^^^^^^^^^^^^^^^
+
+* api - placeholders to automatically generate the api documentation
+
+* devel - documentation for contributing to Matplotlib
+
+* project - about Matplotlib, e.g. mission, code of conduct, licenses, history, etc.
+
+* users - the user documentation, e.g., installation, plotting tutorials,
+configuration tips, faq, explanations, etc.
+
+* thirdpartypackages - redirect to <https://matplotlib.org/mpl-third-party/>
+
+Build Folders
+^^^^^^^^^^^^^
 
 * _static - used by the sphinx build system
 
 * _templates - used by the sphinx build system
 
 * sphinxext - Sphinx extensions for the Matplotlib docs
+
+Symlinks
+--------
+
+During the documentation build, sphinx-gallery creates symlinks from the source folders
+in `/galleries` to target_folders in '/doc'; therefore ensure that you are editing the
+real files rather than the symbolic links:
+
```

Comment:
I find the table format quite noisy. Would you consider a simple table or a bullet list? If you rather keep the fully outlined table, please separate header and content using = signs.